doc/man3/ASYNC_start_job.pod

```diff
@@ -311,6 +330,8 @@ The following example demonstrates how to use most of the core async APIs:
          ASYNC_WAIT_CTX_get_all_fds(ctx, &waitfd, &numfds);
          FD_ZERO(&waitfdset);
          FD_SET(waitfd, &waitfdset);
+
+         /* Wait for the job to be ready for wakeup */
```

Comment:
In this demo we know we will never really wait because `jobfunc` above immediately set the wait signal on `waitfd` rather than waiting for some external event to occur.

Reply:
That's precisely the issue with the documentation. As a user I need examples that explain what happens when I use real world code. Not examples that show how to burn cycles doing nothing.